tests/pipelines/eks/awscli-cl2-load-with-addons.yaml

```diff
@@ -18,6 +16,10 @@ spec:
   - name: slack-message
   - name: amp-workspace-id
     default: ""
```

Comment:
why do we need this `default` ?

Reply:
we don't need it..we can get rid of it..I kept it initially to make so that it doesn't break the existing runs of our pipelines..but since now we are going to anyway update it we can get rid of it.